managed/src/main/resources/swagger.json

```diff
@@ -16,17 +16,14 @@
           "type" : "string"
         },
         "hostVpcId" : {
-          "readOnly" : true,
```

Comment:
Why all these changes to readOnly property?

Reply:
Seems like new swagger plugin does not respect accessMode = READ_ONLY for some reason.
Will dig into that.

Reply:
Huh, found the root cause finally and fixed.
Described the fix in SwaggerGenTest class comment - really tricky issue.

Reply:
Adding issue and fix description here as well for reference:

`
    // So, this one is really tricky. The only play2.8 compatible swagger library I've found,
    // with the same functionality as our previous library, is
    // https://github.com/dwickern/swagger-play. It works fine, except that readOnly fields are
    // not serialized in the resulting swagger.json.
    // The reason is in this "workaround" for the issue with readOnly field they faced:
    // https://github.com/dwickern/swagger-play/commit/b04e4a2e145a277592d29c09d482ccb92138fa14#diff-a74181fef5ee291b8ecaf4773e3e4df1e335e0f0f3640b51f60298e00a143ba2
    // Basically, in marks AbstractProperty.readOnly() method with @JsonIgnore annotation.
    // At the same time - jackson-module-scala adds ScalaAnnotationIntrospector
    // to Swagger ObjectMapper, which makes it treat readOnly() method as getter for readOnly field.
    // As a result - jackson finds all getters/setters for readOnly field across the whole type
    // hierarchy, and treats this field as @JsonIgnore as a result of that.
    //
    // Tried to undo the above "workaround" - but this really causes InvalidDefinitionException
    // during serialization. Decided to not dig into it further to find a better fix for this
    // serialization cycle issue and instead replaced Swagger ObjectMapper with our default mapper
    // with ReferenceSerializationConfigurer.serializeAsComputedRef, which swagger itself adds by
    // default.
`